src/common/active_window.rs

```diff
@@ -4,6 +4,7 @@ use super::window_position::WindowPosition;
 pub struct ActiveWindow {
     pub title: String,
     pub process_name: String,
```

Comment:
`process_path` would be a better name now I think

Reply:
changed in 4016baa